examples/go/tagging/handler/handler.go

```diff
@@ -31,140 +32,145 @@ type vcConfig struct {
 }
 
 // The structure for a Cloud Event.
-type vmEvent struct {
-	Event event `json:"data"`
-}
-type event struct {
-	Vm vmEventArgument `json:"Vm"`
-}
-type vmEventArgument struct {
-	Vm managedObjectReference `json:"Vm"`
-}
-type managedObjectReference struct {
-	Type  string `json:"Type"`
-	Value string `json:"Value"`
+type incoming struct {
+	Data types.Event `json:"data,omitempty"`
 }
 
+var (
+	client *vsClient.Client
+	ctx    = context.Background()
+	cfg    vcConfig
+	sigCh  = make(chan os.Signal)
+)
+
 // Handle a function invocation
 func Handle(req handler.Request) (handler.Response, error) {
-	ctx := context.Background()
-
-	// Retrieve vCenter and tag info from the config file.
-	cfg, err := loadTomlCfg()
-	if err != nil {
-		log.Error(err)
-		return
+	// Connect to vSphere govmomi API if not already done. Also, load the required
+	// vcconfig.
+	if client == nil {
+		err := vsConnect()
+		if err != nil {
+			log.Fatalf("connect to vSphere failed: %v", err)
```

Comment:
It looks like redundant text, but the verbs are different. For `log.Fatalf()` or `log.Printf()`, the verb is `%v`. But for wrapping the message into the error with `fmt.Errorf()`, the verb is `%w`. I'm not sure if it's worth creating another variable to save the message before the verb, though I can be convinced otherwise.